### DISCUSSION TITLE
FATA[0001] failed to create shim task

### DISCUSSION THREAD

**Discussion by imyunha** [containerd/nerdctl#2660] [Q&A]
Hello, everyone. I'm trying to create **kata-containers** using `nerdctl`.

But when I run `sudo nerdctl run -it --runtime io.containerd.run.kata.v2 docker.io/library/ubuntu:22.04 /bin/bash`, error occured. 

`WARN[0000] cannot set cgroup manager to "systemd" for runtime "io.containerd.run.kata.v2"
FATA[0001] failed to create shim task: exit status 1: stdout: , stderr: time="2023-11-25T23:31:24+09:00" level=fatal msg="stat : no such file or directory"
Failed to write to log, write /var/lib/nerdctl/1935db59/containers/default/956479eabad2a4b822655239e7979feab29b7a3162fa815b4eedefa75654d6a8/oci-hook.createRuntime.log: file already closed
: unknown`

If I create **runc** container with `sudo nerdctl run -it docker.io/library/ubuntu:22.04 /bin/bash,` it works.

What's the problem?
Any kind of feedbacks will be very helpful for me.

**Comment by Marshal1l** [comment]
Have you solve this problem? I am same with you now.
